worker/docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
     - IEXEC_WORKER_DOCKER_REGISTRY_ADDRESS_1=${REGISTRY_ADDRESS:-}
     - IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_1=${REGISTRY_USERNAME:-}
     - IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_1=${REGISTRY_PASSWORD:-}
+    - MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE=health,prometheus
```

Comment:
Should we comment this to avoid enabling Prometheus metrics if the user does not want/need them ?

Reply:
The README mentions the URLs /actuator/health and /metrics, I think we should let it as is.

Reply:
not so much of a use, you are right.